fares-data-build-tool/create-data
# feat: cfd-369 delete time restrictions

# Description

-   Allow the user to delete a time restriction.

# Testing instructions

-   Go through the normal user journey for creating NeTEx, when asked if there are time restrictions, say Yes and fill in the details and remember to give it a name.
- Now go to global settings, your time restriction should be listed there.
- Click delete and make sure the delete functionality works.

# Type of change
